src/socean.ts

```diff
@@ -592,6 +594,64 @@ export class Socean {
       transient,
     };
   }
+
+  /**
+   * Register an onAccountChange websocket listener for the stake pool account
+   * and returns the websocket clientSubscriptionId
+   * To remove, call this.config.connection.removeAccountChangeListener(clientSubscriptionId)
+   * @param callback
+   * @param commitment
+   * @returns clientSubscriptionId
+   */
+  onStakePoolChange(
+    callback: (stakePoolAccount: StakePoolAccount, context: Context) => void,
+    commitment?: Commitment,
+  ): number {
+    return this.config.connection.onAccountChange(
+      this.config.stakePoolAccountPubkey,
+      (account, context) => {
+        const stakePoolAccount = getStakePoolFromAccountInfo(
+          this.config.stakePoolAccountPubkey,
+          account,
+        );
+        callback(stakePoolAccount, context);
+      },
+      commitment,
+    );
+  }
+
+  /**
+   * Register an onAccountChange websocket listener for the validator list account
+   * and returns the websocket clientSubscriptionId
+   * To remove, call this.config.connection.removeAccountChangeListener(clientSubscriptionId)
+   * @param callback
+   * @param commitment
+   * @returns clientSubscriptionId
+   */
+  async onValidatorListChange(
+    callback: (
+      validatorListAccount: ValidatorListAccount,
+      context: Context,
+    ) => void,
+    commitment?: Commitment,
+  ): Promise<number> {
+    const {
+      account: {
+        data: { validatorList },
+      },
+    } = await this.getStakePoolAccount();
```

Comment:
Will this always return an object with validatorList? No err/undefined?

Reply:
ah good catch i should probably document that it throws the same errors as `getStakePoolAccount`